Mathlib/Algebra/MvPolynomial/Degrees.lean

```diff
@@ -522,6 +522,10 @@ theorem totalDegree_rename_le (f : σ → τ) (p : MvPolynomial σ R) :
     rcases h' with ⟨s, hs, rfl⟩
     exact (sum_mapDomain_index (fun _ => rfl) (fun _ _ _ => rfl)).trans_le (le_totalDegree hs)
 
+lemma totalDegree_renameEquiv (f : σ ≃ τ) (p : MvPolynomial σ R) :
```

Comment:
Should it be a `@[simp]`?

Reply:
This cannot be because `(renameEquiv R f p)` is not in simp normal form